packages/in_app_purchase/lib/src/billing_client_wrappers/purchase_wrapper.dart

```diff
@@ -89,11 +104,87 @@ class PurchaseWrapper {
   /// Note though that verifying a purchase locally is inherently insecure (see
   /// the article for more details).
   final String originalJson;
+
+  /// The payload specified by the developer when the purchase was acknowledged or consumed.
+  final String developerPayload;
+
+  /// Whether the purchase has been acknowledged.
+  final bool isAcknowledged;
+
+  /// The state of purchase.
```

Comment:
nit: Is there any more information that could be put here besides this? This is basically rewording the variable name.

https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-useless-documentation

Reply:
can't think of any directly related to the purchase state. I added a link to acknowledgePurchase.